### PR TITLE
add workflows to admin

### PR DESCRIPTION
**Pull request by MJedr**

### CODE REVIEW

File: backoffice/workflows/urls.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
-from django.contrib import admin
 from django.urls import include, path
 
 urlpatterns = [
     path("workflows/", include("workflows.urls")),
-    path("admin/", admin.site.urls),
```
Comment (drjova) [inline backoffice/workflows/urls.py:6]: can we keep both?

Comment (MJedr) [reply backoffice/workflows/urls.py:6]: yep, but not sure if we need both - do we need extra admin url on `backoffice.inspire.net/workflows/backoffice`?

Comment (drjova) [reply backoffice/workflows/urls.py:6]: I mean, backoffice.inspire.net/dashboard and backoffice.inspire.net/admin

Comment (MJedr) [reply backoffice/workflows/urls.py:6]: yes, so it's defined in `config.urls`, this adds url `backoffice.net/workflows/admin` which we don't need

**Pull request merged**